openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -29,7 +29,7 @@
 
   .ui-staff__title {
     @extend %t-heading;
-    color: $copy-staff-color;
+    color: $heading-staff-color;
```

Comment:
The staff area had a lot of gray text that wasn't accessible. It's now pretty much all white-- a bit stark looking, but works.

Before:
![image](https://cloud.githubusercontent.com/assets/484484/20154025/86c73b0e-a693-11e6-9f72-d7099563cdaa.png)

After:
![image](https://cloud.githubusercontent.com/assets/484484/20154052/a57e7378-a693-11e6-8cdd-73eb8ec30207.png)